src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/NetCorePackageReferenceProject.cs

```diff
@@ -298,11 +314,32 @@ private static IEnumerable<PackageReference> GetPackageReferences(IEnumerable<Li
                     var existingReference = result.Reference;
                     await existingReference.Metadata.SetPropertyValueAsync("Version", formattedRange);
                 }
+
+                if (installationContext.SuppressParent != LibraryIncludeFlagUtils.DefaultSuppressParent &&
```

Comment:
also, worth refactoring this out into a static helper method.

Reply:
not really, these are simple conditions and make sense to have them right there otherwise we always need to go to another place to figure out the conditions.